nsis/src/nsis/format/NsisScriptHeader.java

```diff
@@ -23,27 +23,33 @@
 		STRUCTURE = new StructureDataType("script_header", 0);
 		STRUCTURE.add(DWORD, DWORD.getLength(), "flags", "First header flags (FH_FLAGS_*)");
 		STRUCTURE.add(STRING, NsisConstants.NSIS_SIGINFO.length, "siginfo", "0xdeadbeef (FH_SIG)");
-		STRUCTURE.add(STRING, NsisConstants.NSIS_MAGIC.length, "nsinst", "NSIS magic (FH_INT1, FH_INT2, FH_INT3)");
-		STRUCTURE.add(DWORD, DWORD.getLength(), "header_size", "points to the header+sections+entries+stringtable in the datablock");
+		STRUCTURE.add(STRING, NsisConstants.NSIS_MAGIC.length, "nsinst",
+				"NSIS magic (FH_INT1, FH_INT2, FH_INT3)");
+		STRUCTURE.add(DWORD, DWORD.getLength(), "header_size",
+				"Points to the header+sections+entries+stringtable in the datablock");
 		STRUCTURE.add(DWORD, DWORD.getLength(), "length_of_following_data",
 				"Length of all the data (including the firstheader and the CRC)");
-		STRUCTURE.add(DWORD, DWORD.getLength(), "compressed_header_size", "If MSB is set following data is compressed");
+		STRUCTURE.add(DWORD, DWORD.getLength(), "compressed_header_size",
+				"If the most significant bit is set, the following data is compressed");
 	}
 
-	public NsisScriptHeader(BinaryReader reader)
-			throws IOException, InvalidFormatException {
+	public NsisScriptHeader(BinaryReader reader) throws IOException, InvalidFormatException {
+		// Values are named after the NSIS implementation of the first header:
```

Comment:
I think this comment belongs in the static init, not here in the constructor? (the fields have the same names as the implementation in STRUCTURE, not here)

Reply:
Good catch!